go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/docker/go v1.5.1-1.0.20160303222718-d30aec9fd63c
 	github.com/fatih/color v1.13.0
 	github.com/foundriesio/go-ecies v0.3.0
+	github.com/karrick/godiff v0.0.2
```

Comment:
i wonder if its worth carrying this whole external dependency? It might be easier to just copy the file with license/attribution comment at the top?

Reply:
We already carry more over 100 dependencies; one more makes no difference. Copying would mean that fioctl will have a larger codebase; not sure it is worth to do that.

FYI the `proxy.golang.org` package manager gives a promise that no package can ever be removed; even after the github project was removed. So, we're safe here.